cmd/bot/send.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
I think we should add this example in a separate pull request, so that we can get the bulk of this change merged in quickly.

I'll provide feedback on this then.